dapp/src/hooks/useAggregatedAcrePointsData.ts

```diff
@@ -20,22 +19,13 @@ type UseAcrePointsReturnType = {
   totalPoolBalance: number
 }
 
-export default function useAcrePoints(): UseAcrePointsReturnType {
+export default function useAggregatedAcrePointsData(): UseAggregatedAcrePointsDataReturnType {
```

Comment:
Yes, I agree that less is more. I think we should split the hook into smaller parts. So that each hook is responsible for only one thing. We would then have independent 3 hooks: `useUserPointsData`, `useAcrePointsData` and `useClaimPoints`. This approach will separate the logic and allow us to use the features from `tanstack` library without any problems.

However, I don't want to do it in this PR and expand the tests. I will do it in https://github.com/thesis/acre/pull/916, because there I will need to check the status of the data fetch. Therefore, let's leave it as it is and the more fixes will come in next PR.